spec/requests/bars_request_spec.rb

```diff
@@ -1,5 +1,150 @@
 require 'rails_helper'
 
 RSpec.describe "Bars", type: :request do
+    let(:user) { User.create email: 'our@test.com', password: '123456', password_confirmation: '123456' }
+    let(:crawl) { Crawl.create title: 'Adventure', user: user }
+
+    # # --index bars--
+    #     it 'gets all the bars' do
+    #         Bar.create
+    #             yelp_id: 'hk496783', 
+    #             name: 'Billys Bar', 
+    #             image_url: 'https://www.picturehere.com/', 
+    #             categories: 'bars all', 
+    #             rating: 3.9, 
+    #             address1: '42 WayBad ave.', 
+    #             city: 'San Clemente', 
+    #             zip_code: '98030', 
+    #             country: 'USA', 
+    #             display_address: '42 WayBad Ave', 
+    #             phone: '+16158429207', 
+    #             display_phone: '1(615)842-9207'
+            
+    #         get '/bars'
+    it 'Adds a bar to a crawl' do
```

Comment:
a better description of this test would be something like it "creates the bar and associates it to the correct crawl"